Meta/check-newlines-at-eof.py

```diff
@@ -0,0 +1,62 @@
+#!/bin/python3
+
+import os
+import subprocess
+import sys
+
+os.chdir(os.path.dirname(__file__) + "/..")
+
+files = subprocess.run(
+    [
+        "git", "ls-files", "--",
+        "*.cpp",
+        "*.h",
+        "*.gml",
+        "*.html",
+        "*.js",
+        "*.css",
+        "*.sh",
+        "*.py",
+        "CMake[^/]*.txt",
+        "*/CMake[^/]*.txt",
```

Comment:
These are not regular expressions. I should use this pattern instead:

```suggestion
        "CMake*.txt",
        "*/CMake*.txt",
```

> An asterisk "*" matches anything except a slash.

https://git-scm.com/docs/gitignore